planning/behavior_path_planner/include/behavior_path_planner/scene_module/pull_over/pull_over_module.hpp

```diff
@@ -111,14 +52,16 @@ enum PathType {
 
 struct PUllOverStatus
 {
+  std::shared_ptr<PullOverPlannerBase> planner{};
   PathWithLaneId path{};
+  PullOverPath pull_over_path{};
+  size_t current_path_idx = 0;
```

Comment:
Other parts uses {} for initialization. So PullOverStatus should also use it.

Reply:
fixed in https://github.com/autowarefoundation/autoware.universe/pull/1911/commits/b47d458c9870cfcb74d01d9866159bd273ee1972